CONTRIBUTING.rst

```diff
@@ -328,6 +334,45 @@ detailing the test coverage::
   networkx/algorithms/approximation/clique.py         42      1     18      1    97%
   ...
 
+Adding tests
+------------
+
+If you're **new to testing**, see existing test files for examples of things to do.
+**Don't let the tests keep you from submitting your contribution!**
+If you're not sure how to do this or are having trouble, submit your pull request
+anyway.
+We will help you create the tests and sort out any kind of problem during code review.
+
+General guidelines for writing good tests:
+
+- doctests always assume ``import networkx as nx`` so don't add that
+- avoid test classes when possible
```

Comment:
For this bullet, maybe something like:

```suggestion
- Prefer pytest fixtures over classes with setup methods.
```

Honestly, this bullet might be more confusing than helpful to new contributors, especially since it sort of contradicts with the "see existing test files for examples of things to do" advice, since the existing tests generally adhere to the class-based organization.